app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss

```diff
@@ -839,6 +893,13 @@ html.omni-nav-v2 {
   }
 }
 
+  /* SEARCH AUTOCOMPLERE RESULTS
```

Comment:
This doesn't need to be under it's own heading at the very bottom. It can go next to all the other general search box styles from lines 53~200. It'd be great if you could add a heading above like 53 that says search styles or something like that.

When you remove this section make sure there's just one empty line between the last CSS rule on line 894 and the comment block on line 903.

Reply:
This sits outside the `#omni-nav-v2` id container, so I don't see anywhere I can nest it inside the current structure
![screen shot 2018-04-10 at 12 01 18 pm](https://user-images.githubusercontent.com/29960336/38577936-441c1e44-3cb7-11e8-8f9a-a4c7b4f27547.png)